src/test/resources/features/scoring.Landing/Grades.feature

```diff
@@ -0,0 +1,167 @@
+@GradesItemList @Regression
+Feature: QuestionBankList
+
+  Background:
+    When user access function website
+    Then user should see login bar
+    When user click login button
+    Then user should see login modal
+    When user login with these credentials
+      | email    | admin@admin.com          |
+      | password | administratorfunctionapp |
+    And user should be in homepage
+
+  @Negative @AccessingGradesPageWithoutLogin
+  Scenario: Accessing Grades Page Without Login
+    Given user should be logged in
+    When user click user section
+    Then user should see dropdown as an indicator of logging in
+    When user click logout button
+    Then user should be in homepage
+    And user should not see grades menu with index 6
+
+  @Positive @AccessingQuestionBankListAsAdmin
+  Scenario: Accessing Question Bank List as an Admin
+    Given user should be logged in
+    And user should see menu bar
+    And user should see grades menu with index 8
+    When user click grades menu with index 8
+    Then user should be in grades page
+    Then user should see grades tab list item with index 0 in grades menu
+    And user should see add button on tab 0
+    And user should see action dropdown button on tab 0 item with index 0
```

Comment:
maybe assert if tab 0 has class `is-active`

Reply:
Assertion is added to step `Then user should see grades tab list item with index 0 in grades menu`
Is this fine? @oliverrsebastian

Reply:
i think it's better to have it as a step in this feature file because we can define the class we wanted to assert instead of hardcoded it in Steps file
cc @DavidWilliamK